tests/integration/test.js

```diff
@@ -59,20 +46,40 @@ describe('Mugshot integration', function() {
         baseline: paths[0],
         screenshot: paths[1],
         diff: paths[2]
-      },
-      wdioInstance, mugshot;
+      };
 
-  before(function() {
-    var options = {
-      desiredCapabilities: {
-        browserName: 'phantomjs'
+function cleanUp() {
```

Comment:
The first and second suite are testing Mugshot with the default rootDirectory. This function is needed to clean that.